### DISCUSSION TITLE
Creating organization, insane captcha puzzle, please help or kill

### DISCUSSION THREAD

**Discussion by martin-plessy** [community/community#61816] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I'm starting to think I actually DO have to be a robot to solve those puzzles:

![image](https://github.com/community/community/assets/48589251/af2985b0-dc86-4bb6-a325-9fb6cf086f4a)

Don't get me wrong, I sure can make the math... But on a timer? That I can't even see? Clicking the button to have another kind of challenge just adds more of them!? If I givE a singlE wrONG ANsWEr, I HaVE TO WaIt uNtiL I coMPLetE alL THe CHaLLenGes to BE reJECtED!!??

EDIT: The bug happens with Brave (private browsing on), but not with Firefox (private browsing on).

**Comment by moonfury-ops** [comment]
Thank you

**Comment by TeemuSo** [comment]
Absolutely this. I can't create organization because this little "infinite data collecting loop". Who the hell would anyways want to automate organization creation, let alone paid organization creation?

**Comment by naveenJ2e** [comment]
Who the hell solve that puzzles not able to create organization.

**Reply by TeemuSo** [reply]
I used my phone and it gave me easier puzzle :D

**Reply by naveenJ2e** [reply]
Thank you bro, it worked.

**Reply by manthanbhadiyadra** [reply]
Thank you. you saved my life after my eyes went dark...tried 30 mins to solve this puzzles on laptop, what the hell

**Comment by DarthJMG2003** [comment]
This issue is still on the page, the worst of all is that i cannot access the option in the mobile version of github, so i cannot create an organization...

**Comment by JaniVilen** [comment]
Designing an organization's intricate CAPTCHA [puzzle](https://sf2apk.com/) is essential for security but can be a challenging balance. Ensuring it's complex enough to deter automated bots, yet user-friendly for real people, requires thoughtful design. Incorporating visual puzzles, logical questions, or user-friendly options can enhance security while preserving user experience.

**Comment by cookouttmenu** [comment]
Yeah, those captcha puzzles can be ridiculously frustrating, it honestly feels like they’re testing your patience more than anything. The trick is not to overthink it; just follow the instructions carefully and look for the most obvious patterns instead of trying to be too clever. Most of the time, they’re simpler than they look. It’s kind of like [letterboxedanswers.net](https://letterboxedanswers.net/), if you rush or guess randomly, you’ll get stuck, but if you stay calm and think step by step, the answers start to make sense.
